modules/main-rules/pages/data-sheets-and-characteristics.adoc

```diff
@@ -57,7 +57,10 @@ Some rules affect some unit types and not others.
 
 Speed:: The distance that you can move the unit in a single movement action (in centimetres).
 It represents a rate at which the unit can move across the battlefield and still be effective when it shoot its weapons.
-You can give units orders that allow you to move units farther than this, but their shooting will suffer.
+You can give units orders that allow you to move units farther than this, but their shooting will suffer. +
++
+Some unit data sheets show a split Speed value such as `20/30 cm`.
+For more about this, see xref:assault-moves-and-charges.adoc[].
```

Comment:
As I said before, **THIS DOES NOT WORK**. 

Players need to know which value to use as the default, and that can't be buried in the Assault moves page. Not that you've explained it there anyway.

If you really dislike 'use the first value for all purposes except Assault moves' as being too specific, we can change it to 'use the first value unless otherwise specified' or similar, but I don't see the benefit if you're going to reference Assault moves anyway.

Reply:
Okay, no need to shout, and I'm not sure what exactly you've said before about this that makes you feel so outraged.

I don't see what the problem is. It's a basic, introductory topic — the main rules are built with progressive disclosure in mind, so they try not to overwhelm the reader with all the whys and wherefores of stuff that comes from special rules, as Split speed values do. (One of the reasons I was trying to think through another approach in that other conversation.) This is normal.

It referenced topic **does** explain it, based on the text that you yourself added but with some edits:

> * Where a unit has a split Speed value such as `20/30 cm`, you may *not* double the first value -- use the second value instead.

However, I can see that we've not explicitly mentioned that the player should use the first value for everything else. That is a valid thing to fix, but again, no shouty shouty needed. If you want things to be perfect in a single pass, in terms of rules and language and structure, with no discussion or iteration required, then you're on a fools errand.

How about something like this?
```suggestion
Some unit data sheets show a split Speed value such as `20/30 cm`.
Use the first value for all movement, except where the rules tell you otherwise.
For information about the second value, see xref:assault-moves-and-charges.adoc[].
```